### DISCUSSION TITLE
post/quickly-start-a-project-with-supabase/

### DISCUSSION THREAD

**Discussion by giscus** [jgjgill/blog#9] [Comments]
# post/quickly-start-a-project-with-supabase/

빠르게 프로젝트를 구축할 수 있는 Supabase를 알아봅니다. Next.js 14, 카카오 Oauth, 데이터베이스에 관한 내용을 다룹니다.

https://jgjgill.com/post/quickly-start-a-project-with-supabase/

**Comment by ShinSeungWoo8987** [comment]
큰  도움 됐어요. 감사합니다.
callback쪽에서 어떻게 처리하는지 확인할수 있도록
깃허브 레포도 연결해두시면 좋을것같아요.
